src/main/resources/templates/admin/email.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<body>
+
+<span th:replace="~{/admin/header :: top}"/>
+
+<a href="/admin/">Home</a> -
+
+<h2>Send Test Email</h2>
+
+<form method="POST" action="/admin/sendEmail">
+    <label for="recipient">Recipient</label>
+    <input type="email" name="recipient" id="recipient" required />
+    <br/>
+    <label for="emailName">Email to Send</label>
+    <select name="emailName" id="emailName">
+        <option th:each="emailName : ${emailNames}"
+                th:text="${emailName}"
+                th:value="${emailName}">
+            Email Name
+        </option>
+    </select>
```

Comment:
This might be simpler as straight HTML since the list of options from the controller is just a list of string literals (as opposed to something like an enum that would give us guaranteed consistency with the valid values on the POST endpoint). Using plain HTML also means it's easy to give the options more human-friendly labels.

Reply:
Good callout, the longer option text is much better.